api_tests/registrations/views/test_registration_detail.py

```diff
@@ -1,297 +1,293 @@
+import pytest
 from urlparse import urlparse
-from nose.tools import *  # flake8: noqa
 from modularodm import Q
 
+from rest_framework import exceptions
 from api.base.settings.defaults import API_BASE
 from website.util import permissions
-
 from osf.models import Registration
 from tests.base import ApiTestCase
```

Comment:
Although, when I tried removing the import, I got an error:

```
AttributeError: 'module' object has no attribute 'ADDONS_AVAILABLE'
```

This is because `init_app` is no longer being called. You can fix this by adding the following to `conftest.py`:

```python
from website.app import init_app
...

# NOTE: autouse so that ADDONS_REQUESTED gets set on website.settings
@pytest.fixture(autouse=True, scope='session')
def app_init():
    init_app(routes=False, set_backends=False)
```